deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -3,7 +3,7 @@ kind: ServiceTelemetry
 metadata:
   name: default
 spec:
-  observabilityStrategy: use_community
+  observabilityStrategy: use_redhat
```

Comment:
I believe this is still safe to use as a default here because it's the CR used if you're deploying from the UI, so it's only used if it's a brand new install.